src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function getVisibleMovie(movies: Movie[], query: string) {
+  const lowerQuery = query.toLowerCase();
+
+  const filteredMovie = movies.filter(movie => (
+    movie.title.toLowerCase().includes(lowerQuery)
+    || movie.description.toLowerCase().includes(lowerQuery)
+  ));
+
+  return filteredMovie;
+}
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = getVisibleMovie([...moviesFromServer], query);
```

Comment:
I think in this case you don't need to copy array with movies, because your source array don't mutate.